### ISSUE TITLE
Doesn't detect that `CustomEvent` is not supported in IE

### ISSUE THREAD

**Issue by haoliangyu** [amilajack/eslint-plugin-compat#279]
According to [MDN](https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent/CustomEvent), `CustomEvent` is not supported in IE. But this is not detected by the plugin.

Example code

```
// it should throw an error here, but it doesn't
const event = new CustomEvent("cat", {
  detail: {
    hazcheeseburger: true
  }
});
window.dispatchEvent(event);
```
